build.proj

```diff
@@ -54,4 +54,23 @@
   <Target Name="Clean">
     <RemoveDir Directories="$(BinDir);$(PackagesDir)" />
   </Target>
+  
+  <Target Name="GenerateCoverageReport"
+    AfterTargets="Build"
+    Inputs="$(CoverageReportDir)\*.coverage.xml"
+    Outputs="$(CoverageReportDir)*.*"
```

Comment:
I suspect the *.* in the Outputs doesn't do what you actually want. If you look at a log you will see that it likely says Outputs "$(CoverageReportDir)*.*" doesn't exist so building the target.